pldoc/static/js/colors.js

```diff
@@ -0,0 +1,132 @@
+define([
+    'jquery'
+], function($) {
+    'use strict';
+
+    var $pane = $('#colors-preview .info-pane'),
+        $title = $($pane).find('.color-info-title'),
+        $desc = $($pane).find('.color-description'),
+        $back = $($pane).find('.color-combinations.background'),
+        $ref = $($pane).find('.color-reference'),
+        $rgb = $($pane).find('.color-rgb'),
+        $hex = $($pane).find('.color-hex');
+
+    var $button;
+
+    var colors;
+
+    var ColorHandler = {
+
+        init: function() {
+            ColorHandler.setupJSONColors();
+            ColorHandler.listenForClick();
+            ColorHandler.listenForKeypress();
+            ColorHandler.listenForClose();
+        },
+
+        setupJSONColors: function() {
+            $.ajax({
+                async: false,
+                url: '/public/js/colors.json',
+                complete: function(data) {
+                    colors = data.responseJSON;
```

Comment:
You can require a JSON file, but only with a [RequireJS Plugin](https://github.com/millermedeiros/requirejs-plugins).